tools/send-to-caas/send-utils.js

```diff
@@ -293,8 +293,28 @@ const isPagePublished = async () => {
   return false;
 };
 
+const getLanguageFirstCountryAndLang = async (path) => {
+  const localeArr = path.split('/');
+  const langStr = (localeArr.length > 1) ? LANGS[localeArr[1]] || LANGS[''] : 'en';
+  let countryStr = (localeArr.length > 2) ? LOCALES[localeArr[2]] || 'xx' : 'xx';
```

Comment:
nit can be simplified a bit
```suggestion
  const langStr = LANGS[localeArr[1]] ?? LANGS[''] ?? 'en';
  let countryStr = LOCALES[localeArr[2]] ?? 'xx';
```

Reply:
Not sure if you want the falsy check instead: `||` vs `??`